website/source/docs/config/index.html.md

```diff
@@ -197,6 +197,17 @@ For Consul, the following options are supported:
 
   * `scheme` (optional) - "http" or "https" for talking to Consul.
 
+  * `check_timeout` (optional) - The check interval used to send health check
+    information to Consul.  Defaults to "5s".
+
+  * `disable_registration` (optional) - If true, then Vault will not register
+    itself with Vault.  If the Consul Agent for Vault and the Consul Servers
+    are older than `0.6.4` it is required to set this to "true" due to API
+    incompatibilities.  Defaults to "false".
```

Comment:
Re: #1, revised the API call because we're never passing more than 1K of output.  There is no longer a dependency on Consul 0.6.4.